tests/testkit_extras/helpers.rs

```diff
@@ -12,80 +12,86 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use bitcoin::util::base58::{FromBase58, ToBase58};
+// FIXME: Sometimes clippy incorrectly calculates lifetimes.
+#![cfg_attr(feature="cargo-clippy", allow(let_and_return))]
+
+use bitcoin::util::base58::ToBase58;
 use serde_json::Value;
 
-use exonum::messages::{Message, RawTransaction};
-use exonum::crypto::{Hash, HexValue};
-use exonum::blockchain::Schema;
-use exonum::storage::StorageValue;
-use exonum::helpers::{self, Height, ValidatorId};
+use exonum::messages::Message;
+use exonum::crypto::HexValue;
+use exonum::blockchain::Transaction;
+use exonum::helpers::{Height, ValidatorId};
 
-use sandbox::sandbox::Sandbox;
-use sandbox::config_updater::TxConfig;
+use exonum_testkit::{TestKit, TestNetworkConfiguration};
 
 use exonum_btc_anchoring::{AnchoringConfig, ANCHORING_SERVICE_NAME};
 use exonum_btc_anchoring::details::btc;
 use exonum_btc_anchoring::details::btc::transactions::{BitcoinTx, RawBitcoinTx, TxFromRaw};
-use exonum_btc_anchoring::details::sandbox::Request;
 use exonum_btc_anchoring::blockchain::dto::{MsgAnchoringSignature, MsgAnchoringUpdateLatest};
 use exonum_btc_anchoring::blockchain::schema::AnchoringSchema;
 
-use {AnchoringSandbox, ANCHORING_VALIDATOR};
+use super::{AnchoringTestKit, TestRequest};
 
 pub use bitcoinrpc::RpcError as JsonRpcError;
 pub use bitcoinrpc::Error as RpcError;
+pub use super::secp256k1_hack::sign_tx_input_with_nonce;
+
+pub fn to_boxed<T: Transaction>(tx: T) -> Box<Transaction> {
```

Comment:
Nitpick: Is this method necessary given that an analogous converter [is already in the `exonum` repository](https://github.com/exonum/exonum/blob/f0f690731b8871533dfd6e2368f03ef3ba182e47/exonum/src/blockchain/service.rs#L465)?